packages/polymath-ui/babel.config.js

```diff
@@ -1,12 +1,7 @@
 const base = require('../../config/babel.config.js');
 
 module.exports = {
-  presets: ['react-app'],
-  plugins: [
-    ...base.plugins,
-    '@babel/plugin-proposal-object-rest-spread',
-    '@babel/plugin-transform-runtime',
-    '@babel/plugin-syntax-async-generators',
-    '@babel/plugin-transform-regenerator',
-  ],
+  babelrcRoots: ['../packages/*'],
+  presets: ['@babel/env', '@babel/flow', '@babel/react'],
+  plugins: [...base.plugins, '@babel/plugin-transform-runtime'],
 };
```

Comment:
This is just part of all the changes I had to do to get this to build. I started adding what we needed for polymath ui from scratch so that we didn't include unnecessary stuff